app/dashboard/models.py

```diff
@@ -1585,6 +1586,12 @@ class Activity(SuperModel):
         on_delete=models.CASCADE,
         blank=True, null=True
     )
+    subscription = models.ForeignKey(
```

Comment:
i think you need to `./manage.py makemigrations` to make a migration for this

Reply:
Definitely. I didn't include it because I thought you guys would wanna do it yourselves or something. But I'm definitely gonna commit the migration then.